Gaijinizer patch/Enemies.txt

```diff
@@ -2739,88 +2739,88 @@ its surroundings until killed once and for all.#Ron
 > BEGIN STRING
 謎が多い
 > CONTEXT: Enemies/612/BookAttribute/2
-There are many mystery#MTLed
+Very Mysterious#Ron
 > END STRING
 
 > BEGIN STRING
 魔界と人間界を介して、様々な世界を渡り歩く行商人。
 > CONTEXT: Enemies/612/BookDescription/4
-Peddler walking across the various worlds through Demon
-Realm and the human realm.#MTLed
+A peddler who travels across worlds via the human
+and demon realms. Unusually for the Oni race, she#Ron
 > END STRING
 
 > BEGIN STRING
 鬼にしては珍しく、戦いを好まず、商売上中立を保つ
 > CONTEXT: Enemies/612/BookDescription/5
-Uncommon for demons, do not like battles, keep business
-neutral#MTLed
+dislikes fighting and prefers to maintain
+neutrality in her business dealings. However,#Ron
 > END STRING
 
 > BEGIN STRING
 だが実際のところは、並々ならぬ実力を持っており。
 > CONTEXT: Enemies/612/BookDescription/6
-But in fact, they have extraordinary abilities.#MTLed
+she actually possesses extraordinary powers#Ron
 > END STRING
 
 > BEGIN STRING
 彼女が『悪魔』の枠に入るのかすら疑問だ。
 > CONTEXT: Enemies/612/BookDescription/7
-It is doubtful that she will fall within the frame of
-demon.#MTLed
+much greater than that of your average demon.#Ron
 > END STRING
 
 > BEGIN STRING
 （パイズリ搾精）
 > CONTEXT: Enemies/613/name
-(Tit-fuck milking)#MTLed
+(Paizuri Milking)#Ron
 > END STRING
 
 > BEGIN STRING
 彼女が必要以上に露出の高い服を着ているのは
 『商品』以外に、代金を支払わせる用意があるためだ。
 > CONTEXT: Enemies/613/BookDescription/3
-She is wearing more exposed clothes than necessary
-Other than "the goods", it is because it is ready to pay
-the price.#MTLed
+Her wearing overly revealing clothes is all part
+of her plan to get clients to pay for more than
+just her "merchandise".#RonLocalizationCheckLater
 > END STRING
 
 > BEGIN STRING
 事実、彼女は性技のテクニックにも淫魔並に精通しており
 > CONTEXT: Enemies/613/BookDescription/5
-In fact, she is as addicted to sex techniques as she is#MTLed
+She is actually well-versed in succubus-level#Ron
+sexual technique. And the paizuri is the position
 > END STRING
```

Comment:
Comment's in the middle here. techinque -> techniques, and remove the "and" here.